source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -199,6 +200,16 @@ data whenever write operations are executed on the collection.
       :start-after: start watch crud example
       :end-before: end watch crud example
 
-
 For a runnable example of the ``watch()`` method using the NodeJS driver, see
 the :ref:`change streams <node-usage-watch>` usage example.
+
+.. _node-retrieve-instruqt-lab:
+
+Interactive Find Operation Lab
+------------------------------
+
+This lab helps you understand how to perform read operations in MongoDB
+by using the ``find()`` method. You do not need to install
+MongoDB or a code editor to complete the lab.
```

Comment:
S: since Vale doesn't like "need," you could reword to something like:
```suggestion
by using the ``find()`` method. You can complete this lab without installing
MongoDB or a code editor.
```

Reply:
changed in all